lib/fog/ovirt/requests/compute/v4/add_volume.rb

```diff
@@ -16,25 +16,23 @@ def add_volume(id, options = {})
           # rubocop:disable Metrics/AbcSize
           def add_options_defaults(options)
             options = options.dup
+            options = convert_string_to_bool(options)
             search = options[:search] || format("datacenter=%<datacenter>s", :datacenter => datacenter)
-            options[:bootable] = options.delete(:bootable) == "true"
+            options[:bootable] = options.delete(:bootable)
```

Comment:
what about if this is nil? the previous behavior would return `false` but the `convert_string_to_bool` method wouldn't add it so now we would be passing nil.

Reply:
As we discussed, The bootable flag doesn't have a default value. if he will receive nill it's the same as false.